### ISSUE TITLE
Allow study-level data access at contrast level

### ISSUE THREAD

**Issue by tsalo** [neurostuff/NiMARE#233]
Add `levels` argument to `Dataset._load_data`, in order to load data from the study level when appropriate (e.g., for metadata and texts, but not for images).

Metadata (and texts, to some extent) can exist over multiple levels for a study. We need to be able to access study-level data at the contrast level.

Stems from #179 and possibly related to #218.

**Comment by tsalo** [comment]
I believe the NIMADS solution was to simply duplicate all study-level metadata at the analysis level. If so, we can close this. @jdkent is that correct?

**Comment by jdkent** [comment]
yes, we now duplicate/inherit the metadata, thanks! closed by #986
